src/index.html

```diff
@@ -15,8 +15,77 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="/"
+        class="logo__img"
```

Comment:
The `href` attribute in anchor tags should not contain spaces. It's recommended to use a path that doesn't include spaces or to encode them if necessary. In this case, the root path `/` is correct, but be mindful of this in other links.